### PR TITLE
fix: fix warning about SUBLANG_NEUTRAL

### PR DESCRIPTION
**Pull request by Montel**

### CODE REVIEW

File: src/rccore/rc_convert.cpp [hunk 1]
```diff
@@ -931,6 +931,8 @@ QString convertLanguageToCode(const QString &name)
                 return "en_US";
             } else if (subLang == "SUBLANG_ENGLISH_ZIMBABWE"_L1) {
                 return "en_ZW";
+            } else if (subLang == "SUBLANG_NEUTRAL"_L1) {
```
Comment (narnaud) [inline src/rccore/rc_convert.cpp:934]: you can remove the if and just keep the else here.

Comment (Montel) [reply src/rccore/rc_convert.cpp:934]: yep but I want to be sure that there is not other sublang
=> with this patch if there is a ENGLISH/other sublang we have a warning

with a "else" we will not see it.

Comment (narnaud) [reply src/rccore/rc_convert.cpp:934]: The complete list is here if you want to be exhausted: https://learn.microsoft.com/en-us/windows/win32/tablet/using-the-recognizers-collection
I missed some of them indeed.

Comment (Montel) [reply src/rccore/rc_convert.cpp]: I updated it.

**Review by narnaud — APPROVED** [review]

**Comment by narnaud** [comment]
Please remove the merge commit (don't know why it's here)

**Pull request merged**